src/Weather/Skysight/SkysightAPI.cpp

```diff
@@ -531,6 +531,18 @@ SkysightAPI::GetData(SkysightCallType t, const TCHAR *const layer,
   return true;
 }
 
+bool
+SkysightAPI::LoadFromFile(Path path, tstring &content)
+{
+  FileLineReader reader(path, Charset::AUTO);
+  content = "";
+  char *line;
+  while ((line = reader.ReadLine()) != NULL)
+    content += line;
+
+  return true;
```

Comment:
What use is this return value?

Reply:
Will introduce maximum. Return true on success, false on exceed limit.

Reply:
Correction. The function which have these lines is not used in anywhere. Will remove.